### ISSUE TITLE
Project Update

### ISSUE THREAD

**Issue by clk125** [JeremyTygh/Norse-Tales#1]
For the first week of the project our group agreed upon the general goal of our project, which is to analyze the characters in Norse Tales based upon how they are described throughout a variety of tales. Each member has selected a handful of tales to read and begin doing basic markup on. While reading we are also thinking about the more specific research questions within our overarching question. One specific question we were able to come up with is "how are characters described before and after being transformed from a human or god form to an animal form, as many of the characters in these tales are animals or humans that became animals. So far we haven't had many issues as we haven't really gotten too far into the project yet. As we move forward we will have more specifics on the who is doing what aspects of the project.

**Comment by djbpitt** [comment]
fake response

**Comment by cngish98** [comment]
My team is still in the process of solidifying our research question(s), so I am impressed that your team already has a more honed-in focus. It seems like an interesting question/topic to pursue! I saw that your group has a minutes file. I think that's a really smart idea - it helps you know exactly what you accomplished in the meeting and makes it easier to track progress. I also think that I may now try to implement that for my own team.

**Comment by peterbussch** [comment]
I agree with the last comment in that your team seems to be quite honed-in and well-organized. I think I may borrow your idea for a minutes folder, just so we can all keep track of what is discussed during a particular meeting and what expectations have been set. Analyzing the human/animal transformations seems really interesting, and I'm excited to see what answers you and your team arrive at.
